### ISSUE TITLE
Clarification on `SBI_SUCCESS` return value from the `sbi_system_suspend` function

### ISSUE THREAD

**Issue by repnop** [riscv-non-isa/riscv-sbi-doc#130]
Page 48 of the v2.0-rc5 document states that `sbi_system_suspend` can return `SBI_SUCCESS`, indicating that "System has suspended and resumed successfully", except that this contradicts the earlier description

> To resume, the hart will jump to supervisor-mode, at the address specified by `resume_addr`, with the specific register values described in Table 52.

which implies that if the function call succeeds, there is no transfer back to the state that the caller was in, instead execution is irreversibly transferred to the address set by `resume_addr` with both of the possible return value registers set to the hart ID and `opaque` value, thus `SBI_SUCCESS` is never observable. Is my interpretation correct here?

**Comment by SiFiveHolland** [comment]
Yes, that is the correct interpretation. The `SBI_SUCCESS` case should be removed from the table.
